x-pack/packages/security-solution/upselling/pages/attack_discovery/page_title/index.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiBetaBadge, EuiFlexGroup, EuiFlexItem, EuiTitle, useEuiTheme } from '@elastic/eui';
+import { css } from '@emotion/react';
+import React from 'react';
+
+import * as i18n from './translations';
+
+const PageTitleComponent: React.FC = () => {
+  const { euiTheme } = useEuiTheme();
+
+  return (
+    <EuiFlexGroup
```

Comment:
It appears that actually flex items there have no effect, since that is getting wrapped in `TruncateText` in the end.

You can see that on the screenshot.
<img width="842" alt="Screenshot 2024-08-05 at 17 25 43" src="https://github.com/user-attachments/assets/76e956f6-1945-4ea2-93a0-25328597b74b">

Can we simplify those styles then?

Reply:
Updated in commit https://github.com/elastic/kibana/pull/188788/commits/22a9ed859867f3c0ff0d796534007d664a781338